org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/desktop/IDesktop.java

```diff
@@ -1038,4 +1038,11 @@ default void removeDataChangeListener(IDataChangeListener listener, Object... da
    */
   NativeNotificationDefaults getNativeNotificationDefaults();
 
+  /**
+   * Special handling for reloading pages via usage of Bookmarks. This can be used to "silently" reload a page without
+   * changing active windows.
+   *
+   * @since 22.0
+   */
+  void refreshOutlineByBookmark(IPage<?> page);
```

Comment:
The fact, that a bookmark is used, is an implementation detail. What about
```suggestion
  void reloadPageFromRoot(IPage<?> page);
```

Reply:
Hey @andibur , good suggestions. I changed it.

Reply:
Perfect, thank you.

Reply:
Perfect, thank you.